pyproject.toml

```diff
@@ -27,6 +27,8 @@ requests = "^2.28.1"
 tqdm = "^4.64.1"
 coverage = "^7.0.1"
 memory-profiler = "^0.61.0"
+edflib-python = "^1.0.6"
+mne-bids = { git = "https://github.com/mne-tools/mne-bids.git", rev = "9bc6fe5" } # until release is available
```

Comment:
The release should arrive soon: https://mne.discourse.group/t/mne-bids-0-13-release-date/7291